benches/nexmark/generator/mod.rs

```diff
@@ -35,6 +35,14 @@ pub struct NexmarkGenerator<R: Rng> {
     wallclock_base_time: Option<u64>,
 }
 
+pub fn wallclock_time() -> Result<u64> {
+    let millis: u64 = SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)?
+        .as_millis()
+        .try_into()?;
```

Comment:
You can probably just use an `as u64` cast here or maybe even just call `.as_secs()` (don't know what behavior you're emulating though)

Reply:
Ah - ok. Updated both and the function to no longer return a `Result`. Thanks.